<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Длина пароля:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lengthInput">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="includeNumbers">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Включить цифры</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="includeUppercase">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Включить заглавные буквы</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="includeSpecial">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Включить спецсимволы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерированный пароль:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="resultOutput">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
